src/EntityFramework.Sqlite/Migrations/SqliteMigrationSqlGenerator.cs

```
@@ -22,6 +22,18 @@ public class SqliteMigrationSqlGenerator : MigrationSqlGenerator
             _sql = sqlGenerator;
         }
 
+        public override void Generate(AddColumnOperation operation, IModel model, SqlBatchBuilder builder)
+        {
+            Check.NotNull(operation, nameof(operation));
+
+            if (operation.DefaultExpression != null
+                && operation.DefaultExpression.Contains("CURRENT_"))
+            {
+                throw new NotSupportedException(Strings.AddColumnWithDefaultExprNotSupported(operation.Table, operation.DefaultExpression));
```

In general, we should only throw if it would result in invalid SQL. Otherwise, we should let the database throw.